x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/additional_toolbar_controls.tsx

```diff
@@ -138,7 +138,11 @@ const AdditionalToolbarControlsComponent = ({
 
   return (
     <EuiFlexGroup alignItems="center" gutterSize="m">
-      {[TableId.alertsOnRuleDetailsPage, TableId.alertsOnAlertsPage].includes(tableType) && (
+      {[
```

Comment:
I wonder if we could use one of the constants we already have (like `DETECTIONS_TABLE_IDS` or `detectionsTimelineIds` maybe?